src/main/webapp/view/transcript/transcript-seeker.js

```diff
@@ -16,7 +16,7 @@
 export default class TranscriptSeeker {
   static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
 
-  #currentTranscriptLine;
+  static #currentTranscriptLine;
```

Comment:
Have a getter instead. Also isn't this static private? How would it be accessible outside the class? (I might be missing something)

Reply:
I don't think we can make things private in js.

Reply:
I think we just can't have private methods but we're allowed to have private variables. The # denotes that it is private.